### PR TITLE
Features/13142 jo

### PR DESCRIPTION
**Pull request by jeffrey-opdam**
Thank you for your pull request!
By completing this pull request, you agree to the [Contributing License Agreement](https://github.com/ALM-Rangers/Team-Services-Bot/blob/master/.github/CLA.md).

Fixes # .

Changes proposed in this pull request:  
- 13142 - Queue a release
- 
- 

@ALM-Rangers/Team-Services-Bot

### CODE REVIEW

File: src/Team-Services-Bot.Api.UnitTests/Services/VstsServiceTest.cs [hunk 1]
```diff
@@ -493,6 +488,42 @@ public class VstsServiceTest
         }
 
         [TestMethod]
+        [SuppressMessage("Microsoft.Design", "CA1024:UsePropertiesWhereAppropriate", Justification = "Test method shouldn't be a property. Test method name corresponds to method under test.")]
+        public async Task GetReleasesTest()
```
Comment (Hussar77) [inline src/Team-Services-Bot.Api.UnitTests/Services/VstsServiceTest.cs]: The test name doesn't match method under test. Should be GetRleaseDefinitoinsAsyncTest.

File: src/Team-Services-Bot.Api/Services/VstsService.cs [hunk 2]
```diff
@@ -306,6 +339,35 @@ public class VstsService : IVstsService
             }
         }
 
+        /// <inheritdoc />
+        public async Task<Release> GetReleaseAsync(string account, string teamProject, int id, OAuthToken token)
```
Comment (Hussar77) [inline src/Team-Services-Bot.Api/Services/VstsService.cs]: Can't find unit test for the GetReleaseAsync()

File: src/Team-Services-Bot.Api.UnitTests/Services/VstsServiceTest.cs (hunk 1 above)
Comment (jeffrey-opdam) [inline src/Team-Services-Bot.Api.UnitTests/Services/VstsServiceTest.cs]: Done.

File: src/Team-Services-Bot.Api/Services/VstsService.cs (hunk 2 above)
Comment (jeffrey-opdam) [inline src/Team-Services-Bot.Api/Services/VstsService.cs]: Yeah missed that one :(

Comment (Hussar77) [inline src/Team-Services-Bot.Api/Services/VstsService.cs]: Excellent, thanks!

**Pull request merged**